src/AlixBa/Addic7edSubtitles/Helpers/Episode.php

```diff
@@ -51,8 +51,8 @@ public function __construct($episodeFilename)
     {
         preg_match($this->pattern, $episodeFilename, $matches);
 
-        $this->showName          = $matches['showname'];
-        $this->sanitizedShowName = self::sanitizeShowName($matches['showname']);
+        $this->showName          = self::capitalizeShowName($matches['showname']);

```

Comment:
I have no opinion on this matter. Would you like me to also remove the "showName" property?